Doppler.Currency/appsettings.json

```diff
@@ -43,9 +43,10 @@
       "CurrencyCode": "MXN",
       "CurrencyName": "Peso Mexicano"
     }
-  },
-  "SlackHook": {
-    "Url": "[SECRET KEY]",
-    "Text": "Doppler Currency Service can't get the currency today, Please check Html format"
   }
+  /* Add Property SlackHook if you want Slack integration
+    "SlackHook": {
+      "Url": "",
+      "Text": ""
+    }*/
```

Comment:
https://github.com/FromDoppler/doppler-currency/pull/16/files#r408872864
https://github.com/FromDoppler/doppler-currency/pull/16/files#r408883612

> I feel that comment in the `appsettings.json` is not a good approach.
> What do you think about leve el `Url` with a empty value as a not configured service?




> I think that it is _more correct_ avoiding the property _SlackHook_ when we do not want to use it. It is more explicit than assuming that behavior when the URL is empty.
> 
> But I am not entirely happy, this comment is a smell that our decision is not entirely well.
> 
> @elsupergomez what do you think a different approach, something like this:
> 
> ```json
>     "SlackHook": {
>       "Enabled": false,
>       "Url": "",
>       "Text": ""
>     }
> ```

In my opinion in prefer that SlackHook node exist and remove the comment, and I prefer to use the URL property as @supergomez said because I don't have to create a secret key in Swarm

Reply:
I added the condition 
`if (!section.Exists() || !Uri.IsWellFormedUriString(slackHookSettings.Url, UriKind.Absolute))
 return new DummySlackHooksService(provider.GetService<ILogger<DummySlackHooksService>>());`
What you do think?